### ISSUE TITLE
not compatible with latest version of npm and node

### ISSUE THREAD

**Issue by jdl2206** [videojs/video.js#6812]
## Description
It looks like it was updated 2 days ago but 

errors when installing:
npm i video.js
npm WARN notsup Unsupported engine for pkcs7@1.0.2: wanted: {"npm":"^1.4.6","node":"^0.10"} (current: {"node":"12.16.3","npm":"6.14.8"})
npm WARN notsup Not compatible with your version of node/npm: pkcs7@1.0.2

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Update to the latest version of npm and node
2.install this plugin
3.

### Expected
Please describe what you expected to see.

### Actual
Please describe what actually happened.

### Error output
If there are any errors at all, please include them here.

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
what version of videojs does this occur with?
#### browsers
what browser are affected?
#### OSes
what platforms (operating systems and devices) are affected?
### plugins
are any videojs plugins being used on the page? If so, please list them below.

**Comment by gkatsev** [comment]
While it's something we should update, it is a warning you can ignore. It shouldn't affect anything.

**Comment by Jegp** [comment]
A more severe variant of this is a complete failure to build videojs on the newest `node:slim` docker. To reproduce

1. `docker run -it node:slim bash`
2. `apt update && apt install git -y`
3. `git clone https://github.com/videojs/video.js.git`
4. `npm install && npm run build`

This runs fine on node v12.18

**Comment by rtrigoso** [comment]
This is not a warning on 13.8.0
```
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for pkcs7@1.0.2: wanted: {"npm":"^1.4.6","node":"^0.10"} (current: {"node":"13.8.0","npm":"6.13.7"})
npm ERR! notsup Not compatible with your version of node/npm: pkcs7@1.0.2
npm ERR! notsup Not compatible with your version of node/npm: pkcs7@1.0.2
npm ERR! notsup Required: {"npm":"^1.4.6","node":"^0.10"}
npm ERR! notsup Actual:   {"npm":"6.13.7","node":"13.8.0"}
```

**Comment by delanni** [comment]
It's as easy as releasing v1.0.3 from pkcs7... the code is there, the version is tagged, but I don't know why it's not released on NPM.

**Comment by gkatsev** [comment]
I'm looking into it. Basically, we never bothered because it's always been just a warning, if it's failing on node 13, definitely needs an update.

**Comment by gkatsev** [comment]
Merged update into VHS main https://github.com/videojs/http-streaming/pull/947.
Will be out soon against 1.x https://github.com/videojs/http-streaming/pull/946 which will come into Video.js shortly there-after.

**Comment by gkatsev** [comment]
The fix is now available in the 7.9.5 pre-release.

**Comment by delanni** [comment]
Guys, you're awesome! Thanks!
